worktree_test.go

```diff
@@ -22,6 +23,7 @@ import (
 	"github.com/go-git/go-git/v5/plumbing/object"
 	"github.com/go-git/go-git/v5/storage/memory"
 
+	"github.com/go-git/go-billy/v5"
```

Comment:
why?

Reply:
The test helper function
```
func setupForRestore(c *C, s *WorktreeSuite) (fs billy.Filesystem, w *Worktree, names []string)  
```
Returns a billy.Filesystem, so I needed to get at the interface.